database/insertData/03_template_E_drugRegistration.js

```diff
@@ -406,7 +781,7 @@ exports.queries = [
                       category: INFORMATION
                     }
                     {
-                      code: "S1Info7"
+                      code: "S1TextPage6"
```

Comment:
The "Shelf life" question (below) -- maybe add the "must be a number" validation to it?